appveyor.yml

```diff
@@ -21,13 +21,20 @@ os: Visual Studio 2015
 
 platform: x64
 
+environment:
+  matrix:
+    - PYTHON_DIR: "C:\\Python27"
+      CMAKE_GENERATOR: "Visual Studio 14 2015"
+    - PYTHON_DIR: "C:\\Python27-x64"
+      CMAKE_GENERATOR: "Visual Studio 14 2015 Win64"
+
```

Comment:
I think we need not specify a generator for CMake if we want to build 32-bit. It is the default generator chosen anyway with MSVC.
It is still good to keep it explicit so readers of this file can know what is being used.